### PR TITLE
Ignore stream type other than Camera in streamUpdated callback

### PR DESCRIPTION
**Pull request by dwecci**

### CODE REVIEW

File: src/app/actions/ConferenceActions.js [hunk 1]
```diff
@@ -1320,24 +1320,13 @@ export class Actions {
...
-          })
-          );
-      });
-
       VoxeetSDK.conference.on("streamUpdated", (user, stream) => {
-        dispatch(
-          ParticipantWaitingActions.onParticipantWaitingUpdated(user.id, stream)
-        );
-        dispatch(this.checkIfUpdateUser(user, stream));
+        if(stream && stream.type=== "Camera") {
```
Comment (Energizz) [inline src/app/actions/ConferenceActions.js:1324]: What if there is update on the screenshare stream ?

Comment (dwecci) [reply src/app/actions/ConferenceActions.js:1326]: I'm not quite sure, just we should not replace camera stream with a screen share stream

Comment (Energizz) [reply src/app/actions/ConferenceActions.js:1326]: It's linked to CCS-1057 right ? I just pushed a fix into the SDK so it might not be usefull no ?

**Comment by Energizz** [comment]
We decided with @dwecci to leave this MR aside, as it was fixed with modification on CCS-1057 (If we have any issue regarding screenshare and streamUpdate, this MR should be investigate again)

**Pull request closed without merging**